docs/tutorials/sosw_layer.rst

```diff
@@ -70,24 +103,96 @@ Or upload it directly via GUI.
 AWS SAM
 ------------------
 
-To deploy Layer with AWS SAM you will simply need to create two files ``samconfig.toml`` and ``template.yaml``
+To deploy layer with AWS SAM you will simply need to create two files ``samconfig.toml`` and ``template.yaml``
 which are represented below.
 
 `samconfig.toml
-<https://link_to_file.com>`_
+<https://github.com/sosw/sosw-examples/blob/master/layers/sosw/samconfig.toml>`_
 
 `template.yaml
-<https://link_to_file.com>`_
+<https://github.com/sosw/sosw-examples/blob/master/layers/sosw/template.yaml>`_
 
 After you create these files you can run them just by entering ``sam build && sam deploy`` in your console.
 
+----------------------------------------------------
+How to connect an AWS Layer to your Lambda Function?
+----------------------------------------------------
+
+Connecting a layer to a Lambda resource in AWS CloudFormation involves specifying the ARN (Amazon Resource Name) of the layer
+when defining the Lambda function. This connection enables the Lambda function to utilize the code and dependencies provided by
+the layer.
+
+Here's a basic CloudFormation template snippet demonstrating how to connect a layer to a Lambda function:
+
+..   code-block:: yaml
+
+    Resources:
```

Comment:
Some extra indent for Resources. And for `code-block` there is an extra space... I don't even think this will render.
Also some indentation problems in the examples below.

Reply:
I'll fix it